docs/README.md

```diff
@@ -0,0 +1,8 @@
+# How to generate this document locally
+
+- Clone this repository and install the required dependencies through poetry.
+- Activate the virtual environment and change your directory to <code>docs</code>
+- Install sphinx and furo through the following commands<br>
+  <code>pip install sphinx</code><br>
```

Comment:
I think we should avoid HTML tags on markdown files whenever is possible. Code tag can be replaced by code markdown blocks. You don't need to use <br> if you use the blocks instead.

Reply:
I updated it. Thank you for the mention.